libraries/chain/proposal_evaluator.cpp

```diff
@@ -81,18 +69,6 @@ struct proposal_operation_hardfork_visitor
    void operator()(const graphene::chain::vesting_balance_create_operation &v) const {
       detail::check_vesting_balance_policy_hf_1268(block_time, v.policy);
    }
-   // hf_199
-   void operator()(const graphene::chain::asset_update_issuer_operation &v) const {
-      if (block_time < HARDFORK_CORE_199_TIME) {
-         FC_ASSERT(false, "Not allowed until hardfork 199");
-      }
-   }
-   // hf_188
-   void operator()(const graphene::chain::asset_claim_pool_operation &v) const {
-      if (block_time < HARDFORK_CORE_188_TIME) {
-         FC_ASSERT(false, "Not allowed until hardfork 188");
-      }
-   }
    // hf_588
    // issue #588
    //
```

Comment:
Please update the rest of this comment to indicate why this can't be moved into `asset_settle_cancel_operation::validate` as suggested.

Reply:
The `HARDFORK_CORE_1468` checks should be removed as well I think.